batch/batch/driver/main.py

```diff
@@ -375,6 +376,30 @@ async def job_started(request, instance):
     return await asyncio.shield(job_started_1(request, instance))
 
 
+@routes.get('/api/v1alpha/worker_wss')
+@active_instances_only
+async def create_driver_worker_websocket_connection(request, instance) -> web.WebSocketResponse:
+    ws = web.WebSocketResponse()
+    await ws.prepare(request)
+    HTTP_ACTIVE_WEBSOCKETS.inc()
+
+    # this is where the back and forth communication actually happens
+    async for msg in ws:
+        if msg.type == aiohttp.WSMsgType.TEXT:
+            if msg.data == 'close':
+                await ws.close()
+            else:
+                print(f"Received message: {msg.data}")
+                await ws.send_str(msg.data + '/answer')
+        elif msg.type == aiohttp.WSMsgType.ERROR:
+            print('ws connection closed with exception %s' % ws.exception())
+
```

Comment:
I think you want a `finally` to close the `ws` and decrement the `HTTP_ACTIVE_WEBSOCKETS`. I'm going off of the code in `auth/auth.py` for `_wait_websocket`.